build/charts/antrea/crds/packetcapture.yaml

```diff
@@ -0,0 +1,194 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: packetcaptures.crd.antrea.io
+  labels:
+    app: antrea
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      additionalPrinterColumns:
+        - jsonPath: .status.phase
+          description: The phase of the PacketCapture.
+          name: Phase
+          type: string
+        - jsonPath: .spec.source.pod
+          description: The name of the source Pod.
+          name: Source-Pod
+          type: string
+          priority: 10
+        - jsonPath: .spec.destination.pod
+          description: The name of the destination Pod.
+          name: Destination-Pod
+          type: string
+          priority: 10
+        - jsonPath: .spec.source.ip
+          description: The IP address of the source.
+          name: Source-IP
+          type: string
+          priority: 10
+        - jsonPath: .spec.destination.ip
+          description: The IP address of the destination.
+          name: Destination-IP
+          type: string
+          priority: 10
```

Comment:
This is not great: showing the Pod name without the namespace is not accurate enough, and we don't handle the destination Service case.

I don't recall what is possible to do with `jsonPath` here, but if the expression language is rich enough, maybe we could just have a `source` and `destination` columns, and populate them correctly based on what was provided by the user.

For example, for `source` we would want the JSONPath expression to evaluate to:
- `.spec.source.pod.namespace + "/" + .spec.source.pod.name` if `.spec.source.pod` is not null
- `.spec.source.ip` otherwise

Not sure if this is supported, but the current printer columns are not ideal IMO.

Reply:
I think it's fine for now. Since `Pod` is a struct, so the results will be like:

```
$ kubectl get pcap -o wide -A
NAME      PHASE     SOURCE-POD                                  DESTINATION-POD                            SOURCE-IP   DESTINATION-IP   TIMEOUT   AGE
pc-test   Running   {"name":"frontend","namespace":"default"}   {"name":"backend","namespace":"default"}                                20        7s
```

For service case as discussed in the other PR before, we don't have a perfect solution for now ( the previous code only work for some cases), so it's removed.

Reply:
Ok, I missed that we were printing the struct

Hopefully this can be improved in the future